src/main/java/org/apache/sling/event/impl/jobs/config/JobManagerConfiguration.java

```diff
@@ -92,6 +92,12 @@ public class JobManagerConfiguration {
             description = "Specify the periodic interval in minutes (default is 48h - use 0 to disable) after which " +
                     "removed jobs (ERROR or DROPPED) should be cleaned from the repository.")
         int cleanup_period() default 2880;
+
+        @AttributeDefinition(name = "Progress Log message's max count",
+                description = "Max number of log messages that can stored by consumer to add information about current state of Job.\n" +
+                        "Any attempt to add more information would result into purging of the least recent messages." +
+                        "default is -1 (to indicate infinite).")
```

Comment:
IIUC then `0` now means "discard any log". If that's true, should this be added to this docu here?

Reply:
Yes, `0` would mean discarding all the log messages. 

I didn't think we need to add this specifically to the documentation because it's not a special case unlike `-1`.

But adding this to the documentation would make this more clear.